src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -21,71 +21,165 @@
+	Channels( Constants.CATEGORY_NETWORK_FEATURES ),
+
+	LevelEmitter( Constants.CATEGORY_NETWORK_BUSES ),
+	CraftingTerminal( Constants.CATEGORY_NETWORK_BUSES ),
+	StorageMonitor( Constants.CATEGORY_NETWORK_BUSES ),
+	P2PTunnel( Constants.CATEGORY_NETWORK_BUSES ),
+	FormationPlane( Constants.CATEGORY_NETWORK_BUSES ),
+	AnnihilationPlane( Constants.CATEGORY_NETWORK_BUSES ),
+	IdentityAnnihilationPlane( Constants.CATEGORY_NETWORK_BUSES ),
+	ImportBus( Constants.CATEGORY_NETWORK_BUSES ),
+	ExportBus( Constants.CATEGORY_NETWORK_BUSES ),
+	StorageBus( Constants.CATEGORY_NETWORK_BUSES ),
+	PartConversionMonitor( Constants.CATEGORY_NETWORK_BUSES ),
+
+	PortableCell( Constants.CATEGORY_PORTABLE_CELL ),
+
+	StorageCells( Constants.CATEGORY_STORAGE ),
+	MEChest( Constants.CATEGORY_STORAGE ),
+	MEDrive( Constants.CATEGORY_STORAGE ),
+	IOPort( Constants.CATEGORY_STORAGE ),
+
+	NetworkTool( Constants.CATEGORY_NETWORK_TOOL ),
+
+	DenseEnergyCells( Constants.CATEGORY_HIGHER_CAPACITY ),
+	DenseCables( Constants.CATEGORY_HIGHER_CAPACITY ),
+
+	P2PTunnelRF( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelME( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelItems( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelRedstone( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelEU( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelLiquids( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelLight( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelOpenComputers( Constants.CATEGORY_P2P_TUNNELS ),
+	P2PTunnelPressure( Constants.CATEGORY_P2P_TUNNELS ),
+
+	MassCannonBlockDamage( Constants.CATEGORY_BLOCK_FEATURES ),
+	TinyTNTBlockDamage( Constants.CATEGORY_BLOCK_FEATURES ),
+
+	Facades( Constants.CATEGORY_FACADES ),
+
+	UnsupportedDeveloperTools( Constants.CATEGORY_MISC, false ),
+	Creative( Constants.CATEGORY_MISC ),
+	GrinderLogging( Constants.CATEGORY_MISC, false ),
+	Logging( Constants.CATEGORY_MISC ),
+	IntegrationLogging( Constants.CATEGORY_MISC, false ),
+	WebsiteRecipes( Constants.CATEGORY_MISC, false ),
+	LogSecurityAudits( Constants.CATEGORY_MISC, false ),
+	Achievements( Constants.CATEGORY_MISC ),
+	UpdateLogging( Constants.CATEGORY_MISC, false ),
+	PacketLogging( Constants.CATEGORY_MISC, false ),
+	CraftingLog( Constants.CATEGORY_MISC, false ),
+	LightDetector( Constants.CATEGORY_MISC ),
+	DebugLogging( Constants.CATEGORY_MISC, false ),
+
+	enableFacadeCrafting( Constants.CATEGORY_CRAFTING ),
+	inWorldSingularity( Constants.CATEGORY_CRAFTING ),
+	inWorldFluix( Constants.CATEGORY_CRAFTING ),
+	inWorldPurification( Constants.CATEGORY_CRAFTING ),
+	InterfaceTerminal( Constants.CATEGORY_CRAFTING ),
+	enableDisassemblyCrafting( Constants.CATEGORY_CRAFTING ),
+
+	AlphaPass( Constants.CATEGORY_RENDERING ), PaintBalls( Constants.CATEGORY_TOOLS ),
+
+	MolecularAssembler( Constants.CATEGORY_CRAFTING_FEATURES ),
+	Patterns( Constants.CATEGORY_CRAFTING_FEATURES ),
+	CraftingCPU( Constants.CATEGORY_CRAFTING_FEATURES ),
+
+	ChunkLoggerTrace( Constants.CATEGORY_COMMANDS, false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)